packages/react-i18n/src/i18n.ts

```diff
@@ -306,6 +306,19 @@ export default class I18n {
 
     return decimal.length === 0 ? DECIMAL_NOT_SUPPORTED : decimal;
   }
+
+  @memoize(
+    (
+      locale: string,
+      options: Intl.DateTimeFormatOptions & {style?: DateStyle} = {},
```

Comment:
I don't think you need the `{style}` typing in here, it is removed before this is ever called